src/quart/app.py

```diff
@@ -776,7 +775,8 @@ def run(
             warnings.warn(
                 f"Additional arguments, {','.join(kwargs.keys())}, are not supported.\n"
                 "They may be supported by Hypercorn, which is the ASGI server Quart "
-                "uses by default. This method is meant for development and debugging."
+                "uses by default. This method is meant for development and debugging.",
+                stacklevel=2,
```

Comment:
Why this change?

Reply:
I made a commit just for this, it was because flake8 was failing (on Windows at least), so I just couldn't commit otherwise :

flake8...................................................................Failed
- hook id: flake8
- exit code: 1

src/quart/app.py:775:13: B028 No explicit stacklevel argument found. The warn method from the warnings module uses a stacklevel of 1 by default. This will only show a stack trace for the line on which the warn method is called. It is therefore recommended to use a stacklevel of 2 or greater to provide more information to the user.